python/dstat-kudu/kudu_dstat.py

```diff
@@ -62,7 +62,7 @@ def start_dstat():
     if operation in ["drop"]:
       drop = True
 
-  client = connect_to("127.0.0.1")
+  client = kudu.connect("127.0.0.1", 7051)
```

Comment:
Never really did, that was just a function in this module that strung it all together (pre-dating the connect method in the client).  If we want to add that, it should probably be in the client itself (note to self).